en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -195,6 +195,7 @@ ArticlesController now::
         public function index()
         {
              $this->set('articles', $this->Articles->find('all'));
+             $this->set(compact('articles'));
```

Comment:
Added the $articles variable as well!



On 22/05/2015 11:37, Mark Story wrote:
>
> In en/tutorials-and-examples/blog/part-two.rst 
> <https://github.com/cakephp/docs/pull/2766#discussion_r30900139>:
>
> > @@ -195,6 +195,7 @@ ArticlesController now::
> >          public function index()
> >          {
> >               $this->set('articles', $this->Articles->find('all'));
> > +             $this->set(compact('articles'));
>
> This will actually cause the code to not work, as there is no 
> |$articles| variable in the local scope.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cakephp/docs/pull/2766/files#r30900139>.
>